### PR TITLE
Added script to create publish_course_modes_to_lms switch

### PR DESCRIPTION
**Pull request by clintonb**
This switch is needed for a proper integration with LMS. Creating it via migration means one less step for setup.

### CODE REVIEW

**Comment by clintonb** [comment]
@rlucioni @bderusha @peter-fogg @jimabramson

**Comment by peter-fogg** [comment]
:+1:

File: ecommerce/extensions/catalogue/tests/test_migrate_course.py [hunk 1]
```diff
@@ -130,7 +130,9 @@ def assert_lms_api_headers(self, request, bearer=False):
 class MigratedCourseTests(CourseMigrationTestMixin, TestCase):
     def setUp(self):
         super(MigratedCourseTests, self).setUp()
-        Switch.objects.get_or_create(name='publish_course_modes_to_lms', active=True)
+        switch = Switch.objects.get_or_create(name='publish_course_modes_to_lms', defaults={'active': True})
```
Comment (rlucioni) [inline ecommerce/extensions/catalogue/tests/test_migrate_course.py]: A tuple is currently being assigned to `switch`, which is causing your test failures. You want `switch, __ = ...`.

File: ecommerce/extensions/api/v2/tests/views/test_courses.py [hunk 2]
```diff
@@ -141,7 +141,11 @@ def test_publish(self):
         path = reverse('api:v2:course-publish', kwargs={'pk': course_id})
 
         # Method should return a 500 if the switch is inactive
-        switch, _created = Switch.objects.get_or_create(name='publish_course_modes_to_lms', active=False)
+        switch, _created = Switch.objects.get_or_create(name='publish_course_modes_to_lms',
```
Comment (rlucioni) [inline ecommerce/extensions/api/v2/tests/views/test_courses.py]: To be consistent, please use `__` instead of `_created`.

**Pull request merged**